src/main/java/uk/ac/ebi/embl/api/validation/check/file/FileValidationCheck.java

```diff
@@ -211,6 +212,37 @@ protected int getAssemblyLevel(ValidationScope scope)
 		return assemblyLevel;
 	}
 
+	public void covid19SeqLengthCheck() throws ValidationEngineException {
+		if (sharedInfo.assemblyType != null && sharedInfo.assemblyType.equalsIgnoreCase(AssemblyType.COVID_19_OUTBREAK.getValue())) {
```

Comment:
- We should add a check that COVID_19_OUTBREAK genomes have taxid 2697049: If they do not we should give the error: 
```
ERROR: Sample organism must be 'Severe acute respiratory syndrome coronavirus 2' (taxid 2697049) for COVID 19 outbreak genomes.
```
- Let's accept the suggested  'covid19SeqLengthCheck' call as the 1st implementation but leave a comment:
```
// TODO: Genome length check should be generic given the tax id for all species with known expected genome length.
```